addplant.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+session_start();
```

Comment:
unsused (all the php on this page) so can remove

Reply:
removed